<?xml version="1.0" ?>
<launch>    
    <param name="gazebo"                   value="true"    type="bool"/>
    <param name="gazebo_robot_name"        value="robotis_op3"/>
    
    <param name="offset_file_path"         value="$(find adol_walking_test_manager)/config/offset.yaml"/>
    <param name="robot_file_path"          value="$(find adol_walking_test_manager)/config/OP3.robot"/>
    <param name="init_file_path"           value="$(find adol_walking_test_manager)/config/dxl_init_OP3.yaml"/>
    <param name="init_pose_file_path"      value="$(find adol_walking_test_manager)/data/ini_pose.yaml"/>
    <param name="device_name"              value="/dev/ttyUSB0"/>
    
    <!-- OP3 Manager -->
    <node pkg="adol_walking_test_manager" type="adol_walking_test_manager" name="adol_walking_test_manager" output="screen"/>
</launch>
